theforeman.org/scripts/debian/add_changelog.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -xe
+
+if [[ $gitrelease == true ]] || [[ -n $pr_number ]]; then
+
+  echo "--Adding changelog entry"
+
+  PACKAGE_NAME=$(head -n1 debian/changelog|awk '{print $1}')
+  DATE=$(date -R)
+  RELEASE="9999-${VERSION}-${suite}+scratchbuild+${BUILD_TIMESTAMP}"
```

Comment:
Where do we set `$suite`? Is it exported somewhere or does it rely on everything being concatted?

Reply:
The latter for right now. This is handled much more cleanly in the pipeline based PR.